### ISSUE TITLE
mta-config's post_install script prints unnecessary output

### ISSUE THREAD

**Issue by dikim33** [oscar-cluster/oscar#214]
**Reported by bli on 25 Jul 2006 05:25 UTC**
During the post_install step, you get the following output from mta-config on SUSE Linux 10.0 with branch-5-0 r5215:

```
POSTFIX is running
#       by SuSEconfig.postfix. So take care that you don't uncomment
queue_directory = /var/spool/postfix
daemon_directory = /usr/lib/postfix
mail_owner = postfix
#mynetworks = hash:/etc/postfix/network_table
#relay_recipient_maps = hash:/etc/postfix/relay_recipients
# "postfix reload" to eliminate the delay.
#header_checks = regexp:/etc/postfix/header_checks
html_directory = /usr/share/doc/packages/postfix/html
sample_directory = /usr/share/doc/packages/postfix/samples
readme_directory = /usr/share/doc/packages/postfix/README_FILES
canonical_maps = hash:/etc/postfix/canonical
virtual_maps = hash:/etc/postfix/virtual
relocated_maps = hash:/etc/postfix/relocated
transport_maps = hash:/etc/postfix/transport
sender_canonical_maps = hash:/etc/postfix/sender_canonical
program_directory = /usr/lib/postfix
smtpd_sender_restrictions = hash:/etc/postfix/access
Postfix is succesfully configured. : SERVER NODE
Shutting down mail service (Postfix)..done
Starting mail service (Postfix)..done
- finished configuring postfix
```

A bunch of the output are printed as a result of the grep call in line 67 of mta-config's [post_install](browser:branches/branch-5-0/packages/mta-config/scripts/post_install) script.  This needs to be fixed.

**Comment by dikim33** [comment]
**Comment by dikim on 2 Aug 2006 23:10 UTC**
I have tried to test this on FC3 and RHEL3-X86 and I could not see this messages. It seems that it occurs on SuSE.

Anyways, I will try to change the line 67 of post_install in the "mta-config" package like this.
```
From
            system("grep postfix $file");

To
            system("grep postfix $file > /dev/null  2>&1");
```

And see if any changes happen to SuSE.

Bernard, if you don't mind, can you please test this line too when you test anything on SuSE?

Thanks,

**Comment by dikim33** [comment]
**Comment by bli on 3 Aug 2006 03:15 UTC**
Yes this works, please check fix into both trunk and branch-5-0.  Thanks!

**Comment by dikim33** [comment]
**Comment by dikim on 3 Aug 2006 03:41 UTC**
(In [5279]) Adding extra codes to redirect output seems to work for SuSE\n So I closes #215 ticket now
